jobs/sentiment-analysis/dbms/sqlserver.py

```diff
@@ -1,29 +1,35 @@
 """Module to connect to the sqlserver database."""
 import pymssql
+from config import _Config
 
 
 class SqlServerConnect:  # pylint: disable=too-few-public-methods
     """Provides the connector to the sqlserver database."""
 
     @staticmethod
-    def connect(app_config):
+    def connect(app_config: _Config):
         """Establishes the connection with sqlsqerver."""
         try:
             connection = pymssql.connect(  # pylint: disable=no-member
                 **app_config.DB_MSSQL_CONFIG
             )
             return connection
-        except Exception as err:
-            raise err
+        except Exception as sqlserver_err:
+            raise sqlserver_err
 
     @staticmethod
-    def get_row_query(
-        primary_keys, input_col, table_name, output_col, limit=100
+    def get_row_query(  # pylint: disable-msg=too-many-arguments
+        primary_keys: str,
+        input_col: str,
+        table_name: str,
+        output_col: str,
+        app_config: _Config,
+        limit: int = 100,
     ) -> str:
         """Returns database specific query for retrieving rows."""
         cols_to_query = f"{primary_keys},{input_col}"
         return (
-            f"select top {limit} {cols_to_query} from {table_name} where "
+            f"select top {limit} {cols_to_query} from {app_config.SCHEMA_NAME}.{table_name} where "
```

Comment:
Do we need a null check here, for cases where you don't need to specify a schema name ?